cmake/scripts/config.csh.in

```diff
@@ -45,7 +45,7 @@ if ( "$same_system" == "1" ) then
   setenv VMCWORKDIR "@VMCWORKDIR@"
   setenv FAIRLIBDIR "@FAIRLIBDIR@"
   setenv PYTHONPATH "@MY_PYTHONPATH@"
-  setenv DYLD_LIBRARY_PATH "@MY_DYLD_LIBRARY_PATH@"
-  setenv LD_LIBRARY_PATH "@MY_LD_LIBRARY_PATH@"
-  setenv PATH "@MY_PATH@"
+  setenv DYLD_LIBRARY_PATH $DYLD_LIBRARY_PATH:"@MY_DYLD_LIBRARY_PATH@"
+  setenv LD_LIBRARY_PATH $LD_LIBRARY_PATH:"@MY_LD_LIBRARY_PATH@"
+  setenv PATH $PATH:"@MY_PATH@"
```

Comment:
IMHO it should be the other way around. `@MY_PATH@` in front of `$PATH`. Same for the library paths.

Reply:
If we're using libraries and binaries that could be shadowed by the system's ones (and that would create a problem), yes...